lasagne/layers/merge.py

```diff
@@ -68,22 +70,57 @@ class ElemwiseSumLayer(MergeLayer):
     before a :class:`DenseLayer`, insert separate :class:`DenseLayer` instances
     of the same number of output units and add them up afterwards. (This avoids
     the copy operations in concatenation, but splits up the dot product.)
```

Comment:
As I said (https://github.com/Lasagne/Lasagne/pull/301#issuecomment-112519950), the `Notes` section should be moved to `ElemwiseSumLayer`, since it only applies to the case of summing. Also the underline is too long, please make it 5 characters (`-----`) while moving it, that would be nice!